internal/pkg/enricher/types.go

```diff
@@ -0,0 +1,427 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package enricher
+
+import "errors"
+
+var (
+	errUnsupportedContainerRuntime = errors.New("unsupported container runtime")
+	errUnsupportedLogLine          = errors.New("unsupported log line")
+)
+
+type auditLine struct {
+	ProcessID    int
+	SystemCallID int
+	Type         string
+	TimestampID  string
+	Executable   string
+}
+
+type containerInfo struct {
+	PodName       string
+	ContainerName string
+	Namespace     string
+	ContainerID   string
+}
+
+// List of x86-x64 syscalls
+// Source: https://raw.githubusercontent.com/torvalds/linux/master/arch/x86/entry/syscalls/syscall_64.tbl
+var systemCalls = map[int]string{
```

Comment:
How portable is this? Since we're using systemCalls as a lookup and don't fail if the syscall cannot be looked up, if there was a syscall not found in this table we would have printed an empty string. Would it be better in that case to print the number as string instead?

Reply:
I changed the output to contain both values (`scid` for syscall id and `scname` for syscall name) - that way we have specific fields for each. As a follow-up PR we could expand support to other architectures when translating the syscall name.